onnxscript/function_libs/torch_lib/ops/linalg.py

```diff
@@ -305,13 +309,78 @@ def aten_linalg_vecdot(x: TensorType, y: TensorType, dim: int = -1) -> TensorTyp
     raise NotImplementedError()
 
 
+@torch_op("aten::linalg_vector_norm", trace_only=True)
 def aten_linalg_vector_norm(
-    self: TensorType,
+    self: TFloat,
     ord: float = 2,
```

Comment:
```suggestion
    ord: float = 2.0,
```

Reply:
This should fix the full graph error

Reply:
Will use XFail.
try INT64 as input.

Reply:
Never mind xfailing, because some cases will pass and thus fail the test